llvm/test/CodeGen/PowerPC/big-endian-store-forward.ll

```diff
@@ -1,12 +1,17 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Should avoid converting tests to generated checks in functional commits

Reply:
If you need to do so, pre-commit regeneration of the tests. Possibly under another PR.

Reply:
> Based on #113338.

which contains the regen-with-UTC. Kindly look only at the second commit.